examples/embed/example_mixedbreadai.py

```diff
@@ -0,0 +1,42 @@
+import os
```

Comment:
We're deprecating the `examples` directory in this repo in favor of keeping all of the docs on our docs page. You can remove this file and we'll add it to the docs once this PR is merged.

- https://docs.unstructured.io/open-source/core-functionality/embedding
- https://github.com/Unstructured-IO/docs
-